src/main/features/response/form/models/statement-of-means/dependants.ts

```diff
@@ -6,32 +6,31 @@ import { toNumberOrUndefined } from 'common/utils/numericUtils'
 import { ValidationErrors as GlobalValidationErrors } from 'forms/validation/validationErrors'
 
 export class ValidationErrors {
-  static readonly UNDER_11_REQUIRED: string = 'Enter a number of children under 11'
-  static readonly BETWEEN_11_AND_15_REQUIRED: string = 'Enter a number of children between 11 and 15'
-  static readonly BETWEEN_16_AND_19_REQUIRED: string = 'Enter a number of children between 19 and 19'
+  static readonly ENTER_AT_LEAST_ONE: string = 'Enter a number for at least one field'
 }
 
 export class Dependants implements Serializable<Dependants> {
 
   @IsDefined({ message: GlobalValidationErrors.YES_NO_REQUIRED })
   hasAnyChildren: boolean
 
-  @ValidateIf(o => o.hasAnyChildren === true)
-  @IsDefined({ message: ValidationErrors.UNDER_11_REQUIRED })
-  @IsInt({ message: GlobalValidationErrors.NUMBER_REQUIRED })
-  @Min(0, { message: GlobalValidationErrors.NUMBER_REQUIRED })
+  @ValidateIf(o => o.hasAnyChildren === true && !o.under11 && !o.between11and15 && !o.between16and19)
+  @IsInt({ message: ValidationErrors.ENTER_AT_LEAST_ONE })
+  crossFieldError: number
```

Comment:
That is not very readable. Alternatively we could put some cross field validation in route.

Reply:
I'm my opinion this is better. You see the model class and you know what are rules. I'd keep it in one place.

What we may consider would be different annotation, eg `@AlwaysError` as in fact we do cross field validation in ValidateIf.

Also can be unit tested easily.

Reply:
I appreciate testability but having `crossFieldError: number` for cross browser validation is not readable at all.